helix-term/src/grammars.rs

```diff
@@ -0,0 +1,290 @@
+use anyhow::{anyhow, Context, Result};
+use std::fs;
+use std::time::SystemTime;
+use std::{
+    path::{Path, PathBuf},
+    process::Command,
+    sync::mpsc::channel,
+};
+
+use helix_core::syntax::{GrammarConfiguration, GrammarSource, DYLIB_EXTENSION};
+
+const BUILD_TARGET: &str = env!("BUILD_TARGET");
+const REMOTE_NAME: &str = "helix-origin";
```

Comment:
This seems a bit weird, since these are all not-Helix repos. Why not just "origin"?